app/src/main/java/com/example/mycleaninglog/dto/extraTask.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.mycleaninglog.dto
+
+import com.google.firebase.firestore.Exclude
+
+data class extraTask(var extraTaskName:String = "", var extraTaskId:String = "", var uniqueID:String = ""){
+
+    override fun toString(): String {
+        return "$extraTaskName"
+    }
+}
```

Comment:
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.